src/lib/journey/_utilities/metadata.utilities.test.ts

```diff
@@ -171,10 +171,11 @@ describe('Test metadata builder function for step', () => {
       ],
     };
     const result = buildStepMetadata(callbackMetadata, stageJson);
-
-    expect(result).toStrictEqual({
+    expect(result).toEqual({
       derived: {
-        isStepSelfSubmittable: false,
+        // because this is a a function now, i'm using the function defined so
+        // the memory is the same and its picked up as such by toEqual.
```

Comment:
I think I know what you mean here, but could you reword and clean up the comment a bit for better clarity?